packages/pycharm-community-bin/PyCharm.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=/usr/share/pixmaps/pycharm.png
```

Comment:
You only need `pycharm` and don't need to specify where icon is.

Reply:
Does the shell look for this if the icon pack doesn't supply a pycharm icon?

Reply:
Shell finds this in `/usr/share/pixmaps` and `/usr/share/icons` for example `/usr/share/applications/element-desktop.desktop` shows `[Desktop Entry]
Name=Element
Exec=/opt/Element/element-desktop %U
Terminal=false
Type=Application
Icon=element-desktop
StartupWMClass=element
Comment=A feature-rich client for Matrix.org
MimeType=x-scheme-handler/element;
Categories=Network;InstantMessaging;Chat;` and the `Icon` is in `/usr/share/icons/hicolor/128x128/apps/element-desktop.png` or Steam's desktop file which shows `Icon=steam` where icon is in `/usr/share/pixmaps/steam.png` so you can also use /usr/share/icons but the current way right now should work (I think).